clusters/build-clusters/01_cluster/sshd-bastion/z/zones_z.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: v1
+data:
+  libvirt-s390x-0-0: |
+    @ 3600 IN SOA coredns (
+        2021040901 ; serial
+        7200 ; refresh (2 hours)
+        3600 ; retry (1 hour)
+        1209600 ; expire (2 weeks)
+        3600 ; minimum (1 hour)
+    )
+    IN NS bastion-z
+
+    *.libvirt-s390x-0-0.  3600  IN A 172.30.93.139
+
+  libvirt-s390x-0-1: |
+    @ 3600 IN SOA coredns (
+        2021040901 ; serial
+        7200 ; refresh (2 hours)
+        3600 ; retry (1 hour)
+        1209600 ; expire (2 weeks)
+        3600 ; minimum (1 hour)
+    )
+    IN NS bastion-z
+
+    *.libvirt-s390x-0-1.  3600  IN A 172.30.12.147
+
+  libvirt-s390x-0-2: |
+    @ 3600 IN SOA coredns (
+        2021040901 ; serial
+        7200 ; refresh (2 hours)
+        3600 ; retry (1 hour)
+        1209600 ; expire (2 weeks)
+        3600 ; minimum (1 hour)
+    )
+    IN NS bastion-z
+
+    *.libvirt-s390x-0-2.  3600  IN A 172.30.226.233
+
+  libvirt-s390x-0-3: |
+    @ 3600 IN SOA coredns (
+        2021040901 ; serial
+        7200 ; refresh (2 hours)
+        3600 ; retry (1 hour)
+        1209600 ; expire (2 weeks)
+        3600 ; minimum (1 hour)
+    )
+    IN NS bastion-z
+
+    *.libvirt-s390x-0-3.  3600  IN A 172.30.227.193
+
+  libvirt-s390x-0-4: |
+    @ 3600 IN SOA coredns (
+        2021040901 ; serial
+        7200 ; refresh (2 hours)
+        3600 ; retry (1 hour)
+        1209600 ; expire (2 weeks)
+        3600 ; minimum (1 hour)
+    )
+    IN NS bastion-z
+
+    *.libvirt-s390x-0-4.  3600  IN A 172.30.226.24
+
+  libvirt-s390x-1-0: |
+    @ 3600 IN SOA coredns (
+        2021040901 ; serial
+        7200 ; refresh (2 hours)
+        3600 ; retry (1 hour)
+        1209600 ; expire (2 weeks)
+        3600 ; minimum (1 hour)
+    )
+    IN NS bastion-z
+
+    *.libvirt-s390x-1-0.  3600  IN A 172.30.109.91
+
+  libvirt-s390x-1-1: |
+    @ 3600 IN SOA coredns (
+        2021040901 ; serial
+        7200 ; refresh (2 hours)
+        3600 ; retry (1 hour)
+        1209600 ; expire (2 weeks)
+        3600 ; minimum (1 hour)
+    )
+    IN NS bastion-z
+
+    *.libvirt-s390x-1-1.  3600  IN A 172.30.107.233
+
+  libvirt-s390x-1-2: |
+    @ 3600 IN SOA coredns (
+        2021040901 ; serial
+        7200 ; refresh (2 hours)
+        3600 ; retry (1 hour)
+        1209600 ; expire (2 weeks)
+        3600 ; minimum (1 hour)
+    )
+    IN NS bastion-z
+
+    *.libvirt-s390x-1-2.  3600  IN A 172.30.162.64
+
+  libvirt-s390x-1-3: |
+    @ 3600 IN SOA coredns (
+        2021040901 ; serial
+        7200 ; refresh (2 hours)
+        3600 ; retry (1 hour)
+        1209600 ; expire (2 weeks)
+        3600 ; minimum (1 hour)
+    )
+    IN NS bastion-z
+
+    *.libvirt-s390x-1-3.  3600  IN A 172.30.206.72
+
+  libvirt-s390x-1-4: |
```

Comment:
Are these just backup in case we need them later?

Reply:
These are for backup incase rewrites doesn't work.